src/Compilers/CSharp/Portable/Parser/LanguageParser_InterpolatedString.cs

```diff
@@ -41,14 +41,11 @@ private ExpressionSyntax ParseInterpolatedStringToken()
             var originalText = originalToken.ValueText; // this is actually the source text
             Debug.Assert(originalText[0] == '$' || originalText[0] == '@');
 
-            var isVerbatim = (originalText[0] == '$' && originalText[1] == '@') ||
-                             (originalText[0] == '@' && originalText[1] == '$');
```

Comment:
we should not have code like thsi that attempted to redo the logic we already have in a centralized location.